frontend/src/routes/(app)/project/[uuid]/[name]/chart/[chartIndex=integer]/+page.svelte

```diff
@@ -61,11 +58,9 @@
 	{:else}
 		<ViewHeader bind:isChartEdit />
 	{/if}
-	{#if chartData}
-		<div class={`flex h-full flex-col overflow-auto pl-2`}>
-			<ChartContainer chartName={chart.name} loading={updatingData}>
-				<svelte:component this={chartMap[chart.type]} {chart} data={chartData} width={900} />
-			</ChartContainer>
-		</div>
-	{/if}
+	<div class={`flex h-full flex-col overflow-auto pl-2`}>
+		<ChartContainer chartName={chart.name} loading={updatingData}>
+			<svelte:component this={chartMap[chart.type]} {chart} data={chartData} width={900} />
+		</ChartContainer>
+	</div>
```

Comment:
If we remove this check, charts can be loaded with empty data, right? Does that make sense?

Reply:
Added in case.